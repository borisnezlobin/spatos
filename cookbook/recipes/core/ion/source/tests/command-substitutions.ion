echo $(echo "one   two    three"    "four 'five' six")
echo 0 "$(echo -e ' one  two  three ')" 1
echo 0 "$(echo -e '  one\ntwo\nthree  ')" 1
echo 0 $(echo -e ' one  two  three ') 1
echo 0 $(echo -e '  one\ntwo\nthree  ') 1
