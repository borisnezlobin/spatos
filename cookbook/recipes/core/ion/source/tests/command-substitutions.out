one   two    three four 'five' six
0  one  two  three  1
0   one
two
three   1
0  one  two  three  1
0   one
two
three   1
